Dockerfile

```diff
@@ -42,13 +51,13 @@ FROM calico/bird:v0.3.3-184-g202a2186-${ARCH} AS calico_bird
 FROM builder AS calico_ctl
 ARG ARCH
 ARG TAG
-RUN git clone --depth=1 https://github.com/projectcalico/calicoctl.git $GOPATH/src/github.com/projectcalico/calicoctl
-WORKDIR $GOPATH/src/github.com/projectcalico/calicoctl
+RUN git clone --depth=1 https://github.com/projectcalico/calico.git $GOPATH/src/github.com/projectcalico/calico
```

Comment:
Instead of cloning it multiple times, what about cloning it twice? One when preparing the builder image and one when preparing the calico-node-builder